adr-work-e76dd272.md

```
@@ -0,0 +1,81 @@
+# ADR-0077: Replace `push_str(&format!(...))` with `write!` in diffguard-core
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)